tools/audio/extraction/audio_extract.py

```diff
@@ -318,3 +449,14 @@ def walk_xmls(out_dict : Dict[int, Tuple[str, Element]], path : str, typename :
         # write the extraction xml if specified
         if write_xml:
             sf.write_extraction_xml(f"assets/xml/audio/soundfonts/{sf.file_name}.xml")
+
+    # ==================================================================================================================
+    # Extract sequences
+    # ==================================================================================================================
+
+    print("Extracting sequences...")
+
+    extract_sequences(audioseq_seg, extracted_dir, version_info, write_xml, sequence_table, sequence_font_table,
+                      sequence_xmls, soundfonts)
+
+    print("Done")
```

Comment:
Maybe `print(f"Sequence extraction took {dt:.3f}s")` (instead of `print("Done")`? The "Done" seems a bit out of place)